front-end/src/app/app-routing.module.ts

```diff
@@ -5,17 +5,18 @@ import { CreatePacienteComponent } from './cruds/pacientes/create-paciente/creat
 import { HomeComponent } from './navegacao/home/home.component';
 import { CreateHospitalComponent } from './cruds/hospitais/create-hospital/create-hospital.component';
 import { LoginComponent } from './acessos/login/login.component';
+import { EditarPacienteComponent } from './cruds/pacientes/editar-paciente/editar-paciente.component';
+import { ListagemPacientesComponent } from './cruds/pacientes/listagem-pacientes/listagem-pacientes.component';
 
 const routes: Routes = [
-  { path: '', redirectTo: '/home', pathMatch: 'full'},
-  { path: 'home', component : HomeComponent },
+  { path: '', redirectTo: '/home', pathMatch: 'full' },
+  { path: 'home', component: HomeComponent },
   { path: 'medico-criar', component: CreateMedicoComponent },
-  { path: 'paciente-criar', component: CreatePacienteComponent},
-  { path: 'hospital-criar', component: CreateHospitalComponent},
+  { path: 'paciente-criar', component: CreatePacienteComponent },
+  { path: 'hospital-criar', component: CreateHospitalComponent },
   { path: 'login', component: LoginComponent },
-
-
-
+  { path: 'editar-paciente/:id', component: EditarPacienteComponent },
+  { path: 'listagem-pacientes', component: ListagemPacientesComponent },
```

Comment:
se n for muito incomodo dona florinda, troca isso pra paciente-editar e paciente-listar